packages/manager/modules/dbaas-logs/src/logs/detail/home/resiliate/resiliate.module.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+
+import '@uirouter/angularjs';
+import 'angular-translate';
+
+import terminate from '../../../../../../billing/src/autoRenew/actions/terminate-agora-service';
```

Comment:
import should be with billing-components instead of referencing the file directly